docs/platform/howto/use-google-private-service-connect.rst

```diff
@@ -0,0 +1,175 @@
+Use Google Private Service Connect with Aiven services |beta|
+=============================================================
+
+Private Service Connect lets you bring your Aiven services into your networks (VPCs) over a private endpoint. The endpoint receives a private IP address from a range assigned by you. Connectivity over the private endpoint is then routed to your Aiven service.
+
+Google Private Service Connect is currently in beta. Please contact support to enable it for your Aiven projects.
+
+Google Private Service Connect is supported for the following services:
+
+* Aiven for Apache Kafka®
+
+For consistency, our tooling calls the feature "privatelink" in all clouds, including Google Cloud.
+
+Prerequisites
+-------------
+
+* :doc:`Aiven CLI </docs/tools/cli>` is installed.
+* The Aiven service is in :doc:`a project VPC </docs/platform/howto/manage-vpc-peering>` in *the same region* where the connecting endpoint will be created. Private Service Connect endpoints are service specific. For each service you wish to connect to, you need to create a separate endpoint.
+
+Set up a Private Link connection
```

Comment:
This is kinda cosmetic, but since our tooling uses "privatelink" without a space, based on what AWS call their feature, I'd also omit the space here so that we have only two names for the thing within this document, not three ("Private Link" with a space is the Azure name for their similar feature)